gradle.properties

```diff
@@ -0,0 +1,14 @@
+useLocal=false
+pluginSinceBuild=241.14494.240
+pluginUntilBuild=242.*
+
+javaVersion=17
+javaTargetVersion=17
+
+# Target IntelliJ Community by default
+platformType=IC
+platformVersion=2024.1
```

Comment:
@staicy123 , please update to latest build of intellij `241.* ` , that is version `2024.1.7`

Reply:
Done